### DISCUSSION TITLE
UCD log - user research entry

### DISCUSSION THREAD

**Discussion by lmilor** [nhsbsa/nhsbsa-digital-playbook#231] [UCD]
The senior URs are currently developing guidance to standardise a format for UR entries in the UCD log

[UR log guidance](https://nhsbsauk-my.sharepoint.com/:w:/g/personal/laumi_nhsbsa_nhs_uk/EYYqfSrjmmlHvB_-z6gq9_ABxitjqMaC0v2HqWEGaJ3frw?e=k7SGAh)

This is currently being reviewed and tested with 3 URs to understand:

- How useful the guidance is
- Whether anything is missing 
- If URs have any difficulties in completing the entry based on the guidance

Will update here when we have some feedback :)

**Comment by lmilor** [comment]
[UR log guidance](https://nhsbsauk-my.sharepoint.com/:w:/g/personal/laumi_nhsbsa_nhs_uk/EZSc3R1jUp5HjEIpZH8nkFUBSs2XlL8i4QfKL0sg9yObDA?e=wEArn1)
This is the second iteration of the guidance following reviewing this with 3 URs. If you have any suggestions for improvement (or spot any typos), please feel free to add a comment to the doc or share here :)

**Comment by lmilor** [comment]
want to clarify with PLs when this is expected to be written - broad agreement that it should be written after a 'block' of UR with those we tested with but this doesn't map onto sprints necessarily

**Comment by nhsbsa-deaco** [comment]
User Research Blank Template - Ready for pilot

[User Research - UCD Log - Blank Template.docx](https://github.com/nhsbsa/nhsbsa-digital-playbook/files/14535033/User.Research.-.UCD.Log.-.Blank.Template.docx)
